torch/_overrides.py

```diff
@@ -546,11 +546,11 @@ def get_testing_overrides():
         torch.quantize_per_channel: lambda input, scales, zero_points, axis, dtype: -1,
         torch.quantize_per_tensor: lambda input, scale, zero_point, dtype: -1,
         torch.quantized_batch_norm: lambda input, weight, bias, mean, var, eps, output_scale, output_zero_point: -1,
-        torch.quantized_gru: lambda data, batch_sizes, hx, params, has_biases, num_layers, dropout, train, bidirectional: -1,
+        # torch.quantized_gru: lambda data, batch_sizes, hx, params, has_biases, num_layers, dropout, train, bidirectional: -1,
         torch.quantized_gru_cell: (lambda input, hx, w_ih, w_hh, b_ih, b_hh, packed_ih, packed_hh, col_offsets_ih,
                                    col_offsets_hh, scale_ih, scale_hh, zero_point_ih, zero_point_hh: -1),
-        torch.quantized_lstm: (lambda input, hx, params, has_biases, num_layers, dropout, train, bidirectional,
-                               batch_first, dtype=None, use_dynamic=False: -1),
+        # torch.quantized_lstm: (lambda input, hx, params, has_biases, num_layers, dropout, train, bidirectional,
+        #                       batch_first, dtype=None, use_dynamic=False: -1),
```

Comment:
@ngoldbaum @ezyang These functions no longer take `Tensor`s for their parameter arguments, which was now tripping up the `test_override` tests as it could not cast `TensorLike` to the appropriate TorchBind object. I removed them from testing for now. Let me know if there's something else we should do here

Reply:
cc @hameerabbasi

Reply:
If they no longer take `Tensor`s, they're no longer supported by `__torch_function__`. This is a backwards incompatible change as these functions can no longer be overridden. May I ask what the motivation was here?

Reply:
These were never tensors, they were typed as Tensor but what it actually was is a C++ struct punned into 1-d ByteTensors. This is making that data type explicit now that we have more rich data types in our type system

Reply:
OK, sounds to me that dropping them from override test is the right thing to do then!